workspaces/spectacle/test/index.ts

```diff
@@ -18,3 +71,40 @@ Tap.test('spectacle batchCommits query', async (test) => {
   });
   test.matchSnapshot(results);
 });
+
+specs.forEach(async (spec) => {
+  Tap.test(`spectacle changelog query ${spec.name}`, async (test) => {
+    const spectacle = await makeSpectacle(OpticEngine, {
```

Comment:
this probably doesn't build anymore. You might have overwritten it but I replaced the usage in this file with InMemoryOpticContextBuilder